src/app/page.tsx

```
@@ -220,13 +241,22 @@ export default function Home() {
         <Alert.Content>
           <Alert.Title>Device restart instructions</Alert.Title>
           <Alert.Description>
-            Once you complete a write operation, you will need to restart your
-            device by pressing and releasing the small “Reset” button near the
-            bottom right, followed quickly by pressing and holding of the main
-            power button for about 3 seconds.
+            {deviceModel === 'x3' ? (
+              <p>
+                Once you complete a write operation, disconnect the USB cable and
```

ESLint

[error] Delete `·and`

(prettier/prettier)